# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/gl/GLContextProviderNull.cpp
# Commit: 16c5dd2cfd60
# Full Hash: 16c5dd2cfd600a62c80ca17cfb35c0868e016056
# Author: Jeff Gilbert <jgilbert@mozilla.com>
# Date: 2020-06-15 21:48:38
# Regressor Bug: 1632249
# File Overlap Count: 3
# Description:
#   Bug 1632249 - Support compositing out-of-process WebGL. r=handyman,lsalzman,nical,geckoview-reviewers,agi,imanol
#   
#   * Majorly simplity CanvasRenderer
#   * Replace GLScreenBuffer with trivial GLSwapChain
#   * Use descriptor structs so that future SharedSurface changes aren't so painful
# ==============================================================================

diff -r ca8139d58d58 -r 16c5dd2cfd60 gfx/gl/GLContextProviderNull.cpp
--- a/gfx/gl/GLContextProviderNull.cpp	Fri Jun 12 21:53:14 2020 +0000
+++ b/gfx/gl/GLContextProviderNull.cpp	Mon Jun 15 18:25:55 2020 +0000
@@ -22,14 +22,14 @@
 }
 
 already_AddRefed<GLContext> GLContextProviderNull::CreateOffscreen(
-    const gfx::IntSize&, const SurfaceCaps&, CreateContextFlags,
+    const gfx::IntSize&, const GLContextCreateDesc&,
     nsACString* const out_failureId) {
   *out_failureId = NS_LITERAL_CSTRING("FEATURE_FAILURE_NULL");
   return nullptr;
 }
 
 already_AddRefed<GLContext> GLContextProviderNull::CreateHeadless(
-    CreateContextFlags, nsACString* const out_failureId) {
+    const GLContextCreateDesc&, nsACString* const out_failureId) {
   *out_failureId = NS_LITERAL_CSTRING("FEATURE_FAILURE_NULL");
   return nullptr;
 }